packages/frontend/src/components/permissioned.tsx

```diff
@@ -34,13 +36,15 @@ export const Permissioned = ({ onBack }: PermissionedProps) => {
                         </Typography>
                         <div className="flex gap-4">
                             <Button
+                                data-testid="back-button"
```

Comment:
Same as above, we could rename this to  `permission-mode-back-button`.

Reply:
agreed